### ISSUE TITLE
Synchronization don't work on Firefox

### ISSUE THREAD

**Issue by wsxy162** [nE0sIghT/chrome-gnome-shell-mirror#87]
Using Firefox's gnome-shell extension with version 10.
My last synchronization marked all extensions being check, and now I have use the same Firefox account on a new OS, but the synchronization doesn't start. Trying install a extension to activate the synchronization, it didn't work.

**Comment by wsxy162** [comment]
FIrefox's own sync has completed, but this extension's Synchronization tab is still a blank.

**Comment by nE0sIghT** [comment]
What did you meant by "blank"? Please attach screenshot.

**Comment by wsxy162** [comment]
I mean the tab has no list any extension installed or synchronized.
![2018-02-24 02-28-11](https://user-images.githubusercontent.com/6559703/36610386-0c367238-190b-11e8-9b28-aa3a2f6ad63d.png)
I'm talking this tab, if you don't get it. "User theme" was installed by my own.

**Comment by nE0sIghT** [comment]
Show me screenshot of this tab from another PC

**Comment by wsxy162** [comment]
It's the same PC, I reinstalled my openSUSE tumbleweed system. I can't provide the previous screenshot.

**Comment by nE0sIghT** [comment]
From [[1]](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/sync):
```
For Firefox a user must have Add-ons checked under the "Sync Settings" options in "about:preferences".
```

Did you had "Add-ons" checked under the "Sync Settings"?

[1] https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/sync

**Comment by nE0sIghT** [comment]
This is fixed in commits a0e55a10 and 7cbd9147.
And will be available in next version.
